src/argparse.lua

```diff
@@ -1069,6 +1109,76 @@ function Parser:get_help()
    return table.concat(blocks, "\n\n")
 end
 
+local function shell_escape(string)
+   return string:gsub("[\\']", "\\%0")
+end
+
+function Parser:get_bash_complete() print "not yet implemented" end
+
+function Parser:get_zsh_complete() print "not yet implemented" end
+
+function Parser:_fish_complete_help(lines, prefix)
+   for _, command in ipairs(self._commands) do
+      for _, alias in ipairs(command._aliases) do
+         local line = ("%s -n '__fish_use_subcommand' -xa '%s\\t\"%s\"'")
+            :format(prefix, alias, shell_escape(command._description))
```

Comment:
Is this the correct way to escape here? within `'` you can't use `'` at all:
```
$ echo '\''
> ^C
$ echo '\'
\
```

Reply:
It's correct for fish. I'll rename and move the function.